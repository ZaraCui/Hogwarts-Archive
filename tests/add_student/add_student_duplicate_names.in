ADD STUDENT Harry Potter
ADD STUDENT Harry Potter
STUDENT 100000
STUDENT 100001
EXIT

