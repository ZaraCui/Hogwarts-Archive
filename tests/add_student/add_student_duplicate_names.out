user: Success.

user: Success.

user: 100000: Harry Potter

user: 100001: Harry Potter

user: Ending Archive process.
